noir-projects/noir-contracts/contracts/test/no_constructor_contract/src/main.nr

```diff
@@ -0,0 +1,44 @@
+use aztec::macros::aztec;
+
+/// Used to test deployment of a contract with no constructor in deploy_method.test.ts and in
+/// private_initialization.test.ts
+#[aztec]
+pub contract NoConstructor {
+    use aztec::{
+        encrypted_logs::log_assembly_strategies::default_aes128::note::encode_and_encrypt_note,
+        macros::{functions::{private, public, utility}, storage::storage},
+        prelude::PrivateMutable,
+    };
+
+    use value_note::value_note::ValueNote;
+
+    #[storage]
+    struct Storage<Context> {
+        private_mutable: PrivateMutable<ValueNote, Context>,
+    }
+
+    /// Arbitrary public method used to test public deployment works for a contract with no constructor.
+    #[public]
+    fn emit_public(value: Field) {
+        context.emit_public_log(/*message=*/ value);
+    }
+
+    /// Arbitrary function used to test that we can call private functions on an uninitialized contract.
```

Comment:
This is a bit misleading, since the contract is not uninitialized - it doesn't have an initializer, and therefore doesn't need initialization. This is closer to the contract being initialized by default.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/13388/commits/803a45d9966f5e74cf9b52b145e1646005c886b6 Thanks